Cargo.toml

```diff
@@ -1,19 +1,20 @@
 [package]
 name = "yaiwr"
-version = "0.0.1"
+version = "0.0.2"
 authors = ["Pavel Durov"]
 edition = "2021"
+rust-version = "1.67.0"
```

Comment:
why?

Reply:
I guess PTSD from broken builds causes by version capabilities.
I can remove it, I don't have strong position on that front

Reply:
It shouldn't be required, and will be annoying in the long-run.

Reply:
ok will remove

Reply:
removed  https://github.com/softdevteam/pavel.yaiwr/pull/7/commits/34fcf6b4f044f68fd22878e390a02b7508e81962